roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -22,6 +22,21 @@
   tags:
     - package-install
 
+- name: make sure ceph_stable_release is set
+  fail:
+    msg: "make sure ceph_stable_release is set to a release name, http://docs.ceph.com/docs/jewel/releases/"
+  when:
+    - ceph_stable_release == 'dummy'
+  tags:
+    - package-install
+
+- name: make sure ceph_stable_release is correct
+  fail:
+    msg: "wrong release name, see http://docs.ceph.com/docs/jewel/releases/"
```

Comment:
If you're wanting to test the ``master`` branch that would be set like ``ceph_dev_branch: master``. The ``ceph_stable_release`` var would still need to be set to the codename of the release that master is currently at.